### PR TITLE
Make live demo more prominent in README.md

### PR DESCRIPTION
**Pull request by joshuahhh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
 
 An accessible, easily internationalizable, mobile-friendly datepicker library for the web
 
+## Live Playground
+
+For examples of the datepicker in action,
+
+Go to [http://airbnb.io/react-dates]
```
Comment (majapw) [inline README.md]: Can you remove these brackets too? :) I don't actually think we need them.

**Comment by majapw** [comment]
Also hi Josh!

**Comment by joshuahhh** [comment]
PTAL @majapw!

(Also hi Maja!)

**Comment by majapw** [comment]
I think once you rebase, we can merge this in! Probably we want this above the `How to use`.

**Comment by joshuahhh** [comment]
Haha I was trying to do this entirely through GitHub's web interface. Guess that's not going to work anymore.

Will do.

**Pull request merged**